presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -296,16 +305,65 @@ public ConnectorSplitSource getSplits(
                 splitSchedulingContext.schedulerUsesHostAddresses(),
                 layout.isPartialAggregationsPushedDown());
 
-        HiveSplitSource splitSource = computeSplitSource(splitSchedulingContext, table, session, hiveSplitLoader);
+        HiveSplitSource splitSource = computeSplitSource(splitSchedulingContext, table, session, hiveSplitLoader, ratio);
         hiveSplitLoader.start(splitSource);
 
         return splitSource;
     }
 
+    private double getSplitScanRatio(
```

Comment:
Can you clarify what splitScanRatio means? The name is confusing to me.  It looks like it's the proportion of bytes we expect to read based on column stats of columns we actually use compared to the total data size.

Reply:
Done